v1/api/swagger/components/schemas/AvailableNetwork.yaml

```diff
@@ -1,5 +1,15 @@
 type: object
 description: Entry of a network available within a data center location
+required: 
+  - ID
+  - Name
+  - LocationID
+  - Description
+  - HostUse
+  - Purpose
```

Comment:
"Purpose" is an optional attribute, so it does not need to be provided. But I'm not clear on what swagger "required" means. Sort of remember it having a different meaning as requiring a non-empty value?

Reply:
All those fields are required so openAPI generator build the GO model with no `omitempty`. This schema is used only for a GET request.